app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/email/SubscribedChangeEmailController.scala

```diff
@@ -76,7 +76,7 @@ class SubscribedChangeEmailController @Inject()(
   ): EitherT[Future, Error, Subscribed] = {
     val journeyWithUpdatedEmail = journey.subscribedDetails.copy(emailAddress = email)
     subscriptionService
-      .updateSubscribedDetails(journeyWithUpdatedEmail)
+      .updateSubscribedDetails(SubscribedUpdateDetails(journeyWithUpdatedEmail, journey.subscribedDetails))
```

Comment:
while your here it might be worth putting in the check to see if the updated details are the same as the old details if it's not too much effort

Reply:
good call, I forgot about that....

Reply:
fixed